internal/pkg/secret/jaas_config_parser.go

```diff
@@ -189,8 +185,7 @@ func (j *JAASParser) parseConfigurationEntry(prefixKey string) (int, int, *prope
 	}
 
 	// Parse Control Flag
-	err = j.parseControlFlag()
-	if err != nil {
+	if err := j.parseControlFlag(); err != nil {
 		return 0, 0, nil, "", err
```

Comment:
Wow, this function returns 5 arguments? 😮

Reply:
Might be something to revisit later for a potential refactor.